akka-actor-typed-tests/src/test/scala/docs/akka/typed/SpawnProtocolDocSpec.scala

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (C) 2014-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.akka.typed
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import scala.concurrent.duration._
+
+import akka.actor.testkit.typed.scaladsl.ActorTestKit
+import akka.actor.typed.TypedAkkaSpecWithShutdown
+import docs.akka.typed.IntroSpec.HelloWorld
+
+//#imports1
+import akka.actor.typed.Behavior
+import akka.actor.typed.SpawnProtocol
+import akka.actor.typed.scaladsl.Behaviors
+
+//#imports1
+
+//#imports2
+import akka.actor.typed.ActorRef
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.Props
+import akka.util.Timeout
+import akka.actor.Scheduler
+
+//#imports2
+
+object SpawnProtocolDocSpec {
+
+  //#main
+  object HelloWorldMain {
+    val main: Behavior[SpawnProtocol] =
+      Behaviors.setup { ctx ⇒
+        // Start initial tasks
+        // ctx.spawn(...)
+
+        SpawnProtocol.behavior
+      }
+  }
```

Comment:
okey that's fine... I had assumed we'd also use it as the default protocol for an ActorSystem without a behavior passed in.